idx/memory/memory.go

```diff
@@ -132,6 +142,9 @@ func (m *MemoryIdx) Load(defs []schema.MetricDefinition) int {
 			continue
 		}
 		m.add(def)
+		// as we are loading the metricDefs from a persistant store, set the lastSave
+		// to the lastUpdate timestamp.
```

Comment:
this comment says exactly what the code does but it should say why it does it, and should explain why this is OK